client/src/modules/shopping-list/index.jsx

```diff
@@ -108,9 +105,8 @@ class ShoppingList extends Component {
       return null;
     }
     const { cohortId, description, isArchived, name } = list;
-    const listItems = items || [];
-    const archivedList = listItems.filter(item => item.isOrdered);
-    const shoppingList = listItems.filter(item => !item.isOrdered);
+    const archivedItems = items ? items.filter(item => item.isOrdered) : [];
```

Comment:
we have cohort and list archivization. And item ordering. Are these the same processes? If they aren't, we should use orderedItems not archiveItems. Or we should discuss this problem...